db/migrations/migrations_test.go

```diff
@@ -256,6 +255,109 @@ func (s *MigrationTestSuite) TestBCDAMigration() {
 				}
 			},
 		},
+		{
+			"Set ACO Termination Details Default",
+			func(t *testing.T) {
+				// create ACO that has been blacklisted by using createTestRow since "Blacklisted"
+				// is no longer an models.ACO property.
+				cmsID := testUtils.RandomHexID()[0:4]
+				acoUUID := uuid.NewUUID()
+				aco := map[string]interface{}{
+					"uuid":        acoUUID,
+					"cms_id":      cmsID,
+					"name":        "Blacklisted ACO",
+					"blacklisted": true,
+				}
+				createTestRow(t, db, "acos", aco)
+
+				defer postgrestest.DeleteACO(t, db, acoUUID)
+
+				migrator.runMigration(t, "11")
+				assertColumnExists(t, true, db, "acos", "termination_details")
+
+				sb := sqlFlavor.NewSelectBuilder()
+				sb.Select("blacklisted", "termination_details").
+					From("acos").
+					Where(
+						sb.Equal("blacklisted", true),
+					)
+
+				query, args := sb.Build()
+				rows, err := db.Query(query, args...)
+				assert.NoError(t, err)
+				defer rows.Close()
+
+				for rows.Next() {
+					var blacklisted bool
+					var termination_details string
```

Comment:
I'm thinking that we would want to use a `sql.NullString` here. And for the check, you can say `termination_details.IsValid()`.

Without this, we would fail at the rows.Scan method.

Reply:
And small nit pick (nit) on naming, can we used mixedCaps here?

Reply:
Fixed